src/__fixtures__/newPortalUser.ts

```diff
@@ -102,7 +102,7 @@ export const exampleCollection2: MySpaceWidget = {
 }
 
 export const newPortalUser = {
-  userId: 'CAMPBELL.BERNADETTE.5244446289',
+  userId: 'BERNADETTE.CAMPBELL.5244446289@testusers.cce.af.mil',
```

Comment:
We had some inconsistencies in our userId test data that I needed to update. If there was a reason for the mismatch, let me know!